### PR TITLE
Handle when readme is null

### PR DESCRIPTION
**Pull request by uhLeeshUh**
This is a first pass at closing #27, but please leave any and all comments (nits welcomed!) as I'm learning python best practices

### CODE REVIEW

File: qri/dataset.py [hunk 1]
```diff
@@ -34,7 +34,7 @@ def render(self):
         return markdown.markdown(self.script)
 
     def __repr__(self):
-        return self.script
+        return self.script or ''
```
Comment (uhLeeshUh) [inline qri/dataset.py:37]: @dustmop I saw you put the handle comment in the `__init__` class but my guess was it's best to handle it here since we're returning the obj representation when asking for a dataset's readme. Am I off track? I also saw in your issue you intended to return `None` but I thought this method needed to return a string so I just made it empty. Open to feedback on how that can be more declarative though

File: tests/client_test.py [hunk 2]
```diff
@@ -19,6 +19,11 @@ def test_client_get(self):
         self.assertEqual(str(ds), expect)
         self.assertEqual(ds.structure.format, 'csv')
 
+    def test_client_get_readme(self):
```
Comment (uhLeeshUh) [inline tests/client_test.py:22]: Happy to move this test into the above one if you don't think it merits its own?

File: qri/dataset.py (hunk 1 above)
Comment (dustmop) [reply qri/dataset.py:37]: Fixing it here looks like the right thing to do, I just wasn't sure where it should go.

However, thinking it over a bit more, I feel it's better to return the actual string `'None'` if the readme doesn't exist. The `__repr__` *must* return a string, but the string should be a representation of the data, not just a stringified version. For example, `__repr__` for the string `"hello"` will contain the outer quotes. Another way to think about this is that the result of `__repr__` should be parsable to get back the original value, which is sometimes stated as `eval(repr(c)) == c`.

We're playing a bit fast and loose with that here, as the readme's repr is just returning its text contents. It should probably have the same type wrapper that Meta and Structure are using.

In summary, I think this is fine for fixing the problem at hand! I'll follow up with another commit to handle the related problems mentioned.

File: tests/client_test.py (hunk 2 above)
Comment (dustmop) [reply tests/client_test.py:22]: It's good right here, I like to put more specific tests (get readme) after the more general ones (get).

**Review by dustmop — APPROVED** [review]
LGTM, merging.
